www/perl-lib/SMT/Mirror/RpmMd.pm

```diff
@@ -643,7 +653,9 @@ sub mirror()
                 {
                     File::Copy::copy( $fullpath, $metatempdir."/$entry" ) or do
                     {
-                        printLog($self->{LOG}, $self->vblevel(), LOG_ERROR, "copy metadata failed: $!");
+                        my $logMessage = "copy metadata failed: $!";
+                        printLog($self->{LOG}, $self->vblevel(), LOG_ERROR, $logMessage);
+                        push(@{$self->{STATISTIC}->{ERROR_MESSAGES}}, $logMessage);
```

Comment:
Could the preceding two lines that are repeated a lot not be pushed into a function?